pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: layout.pug
+navigationTitle: Konvoy 2.1.4 Release Notes
+title: Konvoy 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Konvoy&reg; version 2.1.2 was released on October dd, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+To get started with Konvoy, [download](../../download/) and [install](../../choose-infrastructure/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+## Fixes and Improvements
+
+<Add or remove>
+
+## Component updates
+
+The following services and service components are upgraded to the listed version:
+
+- centralized-grafana: 18.1.3
+- centralized-kubecost: 0.20.0
+- cert-manager: 0.2.7
+- dex: 2.9.10
+- external-dns: 2.20.5
+- fluent-bit: 0.16.2
+- gatekeeper: 0.6.9
+- grafana-logging: 6.16.14
+- grafana-loki: 0.33.1
+- istio: 1.9.2
+- jaeger: 2.21.0
+- karma: 2.0.0
+- kiali: 1.29.1
+- knative: 0.3.9
+- kube-oidc-proxy: 0.2.5
+- kube-prometheus-stack: 18.1.3
+- kubecost: 0.20.0
+- kubefed: 0.9.0
+- kubernetes-dashboard: 5.0.2
+- kubetunnel: 0.0.8
+- logging-operator: 3.15.0
+- metallb: 0.12.2
+- minio-operator: 4.1.7
+- nfs-server-provisioner: 0.6.0
+- nvidia: 0.4.3
+- project-grafana-logging: 6.16.14
+- project-grafana-loki: 0.33.1
+- project-logging: 1.0.0
+- prometheus-adapter: 2.11.1
+- reloader: 0.0.99
+- thanos: 0.4.5
+- traefik: 10.3.0
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.5
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### No apparent upgrade path for 1.X vSphere clusters to 2.X
+
+There is a way to migrate existing vsphere clusters from 1.8 to 2.1. When attempting, this ends in failure due to providerID issues with the migration.
+
+<Need to validate if we are still keeping the following issues from 2.1.3:>
```

Comment:
Validate issues below - keep or remove these for 2.1.4?

Reply:
Keep them, there's no fix for this until 2.2.1, as mentioned in the body of the workaround.

Reply:
these are all for Kommander though. So I'd take them out of these release notes, but add them to the Kommander release notes for 2.1.4.